apps/web/src/services/administration/iam/user/modules/user-management-modal/modules/NotificationEmailForm.vue

```diff
@@ -162,7 +162,7 @@ const handleClickSend = async () => {
         emit('change-verify', true);
 
         if (state.loginUserId === props.item.user_id) {
-            await store.dispatch('user/setUser', { email: email.value });
+            await store.dispatch('user/setUser', { email: email.value, emailVerified: true });
```

Comment:
question: Isn't it `email_verified`? 🧐

Reply:
Is it better to rename them to the same name, even though one is a column name for incoming data and the other is a value intended for use in Vuex?